ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -130,7 +130,7 @@ - (void)validatedAcquireToken:(ADAuthenticationCallback)completionBlock
         }
     }
     
-    [self requestToken:completionBlock];
+    [self requestToken:error completionBlock:completionBlock];
```

Comment:
The place where we're really looking for underlying errors is when the server sent us an OAuth error and we end up stomping on it and returning interaction required instead.